### DISCUSSION TITLE
Usage of hinfsyn

### DISCUSSION THREAD

**Discussion by bnavigator** [python-control/python-control#597] [Q&A]
Thank you very much for your advise, it solved my problem very well. Another problem trapped me. When I use the hinfsyn function, the internal command runs to sb10ad (out = sb10ad(n, m, np_, ncon, nmeas, gamma, P.A, P.B, P.C, P.D)), debugging cannot be performed,  the console does not display content,  and there is no error message.

using command control.hinfsyn(P,1,1).
P is calculated using the control.augw() command.
 slycot.__version__'0.4.0.0'
control.__version__'0.9.0'
Pycharm in window system.

_Originally posted by @JinmingChe in https://github.com/python-control/python-control/issues/550#issuecomment-809427604_

**Comment by bnavigator** [comment]
So what's the question or problem exactly?

**Reply by JinmingChe** [reply]
My question looks the same as this issue https://github.com/python-control/python-control/issues/367#issue-546152816.

**Comment by JinmingChe** [comment]
Thank your for your remind.
The question is when I use the hinfsyn function, it block in "out = sb10ad(n, m, np_, ncon, nmeas, gamma, P.A, P.B, P.C, P.D)" command. I think there may be a problem with sb10ad implement. The program did not report an error.  So, How can I debug?
